assets/app/view/home.rb

```diff
@@ -21,6 +21,13 @@ def render
       your_games, other_games = @games.partition { |game| user_in_game?(@user, game) }
       grouped = other_games.group_by { |game| game['status'] }
 
+      # Ready, then active, then unstarted.
+      your_games.sort_by! do |game|
+        [user_is_acting?(@user, game) ? -2 : 2,
```

Comment:
can you reformat this to be 
[
  x,
  y,
  z,
]

also is it intentional to sort older games first as opposed to new games?

Reply:
No, that was an oversight, thanks. Fixed now.